File: src/main/java/net/kenro/ji/jin/purescript/psi/impl/PSIdentInfixImpl.java
package net.kenro.ji.jin.purescript.psi.impl;

import com.intellij.lang.ASTNode;
import com.intellij.psi.PsiElementVisitor;
import net.kenro.ji.jin.purescript.psi.PSIdentInfix;
import net.kenro.ji.jin.purescript.psi.PSVisitor;
import org.jetbrains.annotations.NotNull;

public class PSIdentInfixImpl extends PSPsiElement implements PSIdentInfix {

    public PSIdentInfixImpl(final ASTNode node) {
        super(node);
    }

}

File: src/main/java/net/kenro/ji/jin/purescript/psi/impl/PSIdentInfixImpl.kt
package net.kenro.ji.jin.purescript.psi.impl;

import com.intellij.lang.ASTNode;
import net.kenro.ji.jin.purescript.psi.PSIdentInfix;

public class PSIdentInfixImpl extends PSPsiElement implements PSIdentInfix {

    public PSIdentInfixImpl(final ASTNode node) {
        super(node);
    }

}
